tests/FifoStockTest.php

```diff
@@ -13,10 +13,9 @@ public function setUp()
 
     public function testTake()
```

Comment:
Teraz mnie trochę olśniło.
Mamy za małą próbkę testu.
Dodajemy 1 element i 1 element wyciągamy.
Skąd mamy mieć pewność czy dostajemy elementy w kolekności FIFO czy LIFO? 
żeby to przetestować musimy mieć co najmniej 2 elementy, a najelpiej gdyby były 3.